packages/relay/src/lib/eth.ts

```diff
@@ -125,19 +125,21 @@ export class EthImpl implements Eth {
    * This method is implemented to always return an empty array. This is in alignment
    * with the behavior of Infura.
    */
-  accounts() {
-    this.logger.trace('accounts()');
+  accounts(requestId?: number) {
+    const requestIdPrefix = requestId ? `[${constants.REQUEST_ID_STRING}${requestId}]` : '';
```

Comment:
You can move this to a private function so instead of repeating the logic you just call getRequestIdMessage(requestId)
```suggestion
    const requestIdPrefix = getRequestIdMessage(requestId);
```

where
```
  private getRequestIdMessage(requestId: string): string {
    return requestId ? `[${constants.REQUEST_ID_STRING}${requestId}]` : '';
  }
```